README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
Clicking "Run" with an empty program gives an unhelpful error message in the sim console:
> Cannot read properties of undefined (reading 'documentElement')

I think it only happens the very first time you use Scratch. After that (when the Scratch program isn't a completely empty string), it shows the more useful "block not found" error.

Dev console shows:
```
dom-parser.js:178 
        
       [xmldom error]	invalid doc source 
@#[line:0,col:undefined]
error @ dom-parser.js:178
DOMParser.parseFromString @ dom-parser.js:27
mod.onRuntimeInitialized @ worker.ts:210
doRun @ VM5:156756
run @ VM5:156773
runCaller @ VM5:156681
removeRunDependency @ VM5:136060
receiveInstance @ VM5:136223
receiveInstantiationResult @ VM5:136241
Promise.then (async)
instantiateArrayBuffer @ VM5:136249
instantiateAsync @ VM5:136293
createWasm @ VM5:136311
eval @ VM5:141483
__WEBPACK_DEFAULT_EXPORT__ @ require.ts:9
(anonymous) @ worker.ts:178
fulfilled @ worker.ts:10
Promise.then (async)
step @ worker.ts:12
fulfilled @ worker.ts:10
Promise.then (async)
step @ worker.ts:12
(anonymous) @ worker.ts:13
__awaiter @ worker.ts:9
startScratch @ worker.ts:164
(anonymous) @ worker.ts:283
(anonymous) @ worker.ts:13
__awaiter @ worker.ts:9
start @ worker.ts:261
ctx.onmessage @ worker.ts:292
worker.ts:242 
        
       TypeError: Cannot read properties of undefined (reading 'documentElement')
    at Ast.parse (Ast.ts:121:1)
    at InstanceIncompleteOptions.complete (Instance.ts:6:1)
    at new Instance (Instance.ts:129:1)
    at mod.onRuntimeInitialized (worker.ts:209:1)
    at doRun (eval at __WEBPACK_DEFAULT_EXPORT__ (require.ts:9:1), <anonymous>:156756:71)
    at run (eval at __WEBPACK_DEFAULT_EXPORT__ (require.ts:9:1), <anonymous>:156773:5)
    at runCaller (eval at __WEBPACK_DEFAULT_EXPORT__ (require.ts:9:1), <anonymous>:156681:19)
    at removeRunDependency (eval at __WEBPACK_DEFAULT_EXPORT__ (require.ts:9:1), <anonymous>:136060:7)
    at receiveInstance (eval at __WEBPACK_DEFAULT_EXPORT__ (require.ts:9:1), <anonymous>:136223:5)
    at receiveInstantiationResult (eval at __WEBPACK_DEFAULT_EXPORT__ (require.ts:9:1), <anonymous>:136241:5)
```